pkgs/development/python-modules/web3/default.nix

```diff
@@ -72,6 +72,10 @@ buildPythonPackage rec {
 
   doCheck = false;
 
+  postPatch = ''
+    substituteInPlace setup.py --replace "types-protobuf==3.19.13" "types-protobuf"
```

Comment:
I think you want `pythonRelaxDeps`

Reply:
thanks! let me try

Reply:
oh but that `pythonRelaxDeps` seem not affecting the `extras_require .linter` in https://github.com/ethereum/web3.py/blob/main/setup.py

I  am not familiar with nix python packaging yet.

Reply:
You're right, it's fine like this, let's move on :)